_posts/2021-12-14-dec-community-call-summary.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: Highlights from the December 2021 Community Call on Constructive Code Review
+tags: [community call, code review, training]
+posted_by: David Nicholson
+---
+
+# Highlights from the December 2021 Community Call
+
+Last week, US-RSE held its monthly community call.
+For a topic, we chose "Conducting Constructive Code Reviews", as
+[suggested on GitHub](https://github.com/USRSE/monthly-community-calls/issues/9#issuecomment-921989398).
+Judging from the number of new faces that registered for the call,
+the US-RSE commmunity is very interested in this topic.
+
+We'd like to share a couple of highlights from the call.
+If these highlights whet your interest,
+then the video is already up on YouTube for you to watch:  
+https://www.youtube.com/watch?v=0vdfNojvlck.
```

Comment:
this is not a link in the preview - and maybe better to put this as a link under "a video is already up" than to spell out the URL in the post.

Reply:
Modified as suggested